Resources/Prototypes/_Goobstation/Catalog/uplink_catalog.yml

```
@@ -87,6 +87,17 @@
   categories:
   - UplinkWeaponry
 
+- type: listing
+  id: UplinkWeaponRifleBurner
+  name: uplink-weapon-burner-name
+  description: uplink-weapon-burner-desc
+  icon: { sprite: /Textures/_Goobstation/Objects/Weapons/Guns/Rifles/burner.rsi, state: icon }
```

yamllint (1.35.1)

[error] too many spaces inside braces

(braces)